consensus/consensus_v2.go

```diff
@@ -268,23 +270,23 @@ func (consensus *Consensus) onPrepare(msg *msg_pb.Message) {
 	// proceed only when the message is not received before
 	_, ok := prepareSigs[validatorAddress]
 	if ok {
-		utils.GetLogInstance().Debug("Already received prepare message from the validator", "validatorAddress", validatorAddress)
+		utils.GetLogger().Debug("Already received prepare message from the validator", "validatorAddress", validatorAddress)
 		return
 	}
 
 	// Check BLS signature for the multi-sig
 	var sign bls.Sign
 	err = sign.Deserialize(prepareSig)
 	if err != nil {
-		utils.GetLogInstance().Error("Failed to deserialize bls signature", "validatorAddress", validatorAddress)
```

Comment:
this morning i was planning to replace all GetLogInstance by GetLogger like this. After talking to EK, i learn that it's not recommended to use GetLogger extensively.

https://github.com/harmony-one/harmony/pull/975 is my closed PR of doing that. Maybe redo it. @astralblue